lib/landrush/cap/host/redhat/install_dnsmasq.rb

```diff
@@ -0,0 +1,14 @@
+module Landrush
+  module Cap
+    module Redhat
+      module InstallDnsmasq
+        class << self
+          def install_dnsmasq(_env)
+            system('sudo yum clean all > /dev/null 2>&1')
+            system('sudo yum install -y -q dnsmasq > /dev/null 2>&1')
+          end
```

Comment:
For Fedora we should use dnf. So we should add another host capability as Fedora which Vagrant provides us.